app-text/master-pdf-editor/master-pdf-editor-5.6.42.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg-utils
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}-qt5.x86_64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+RDEPEND="
+	>=dev-qt/qtcore-5.9.8:5
+	>=dev-qt/qtgui-5.9.8:5
+	>=dev-qt/qtnetwork-5.9.8:5
+	>=dev-qt/qtprintsupport-5.9.8:5
+	>=dev-qt/qtsvg-5.9.8:5
+	>=media-gfx/sane-backends-1.0
```

Comment:
Maybe adding sys-apps/dbus [(libdbus-1.so.3: cannot open shared object file: No such file or directory)](https://740212.bugs.gentoo.org/attachment.cgi?id=658624) could help solving bug 740212?

Reply:
I have replied to you on the bug report. Did you try the new ebuild or the one with your proposed change that I added to the bug report?